snyk-sdk/src/main/java/io/snyk/sdk/api/v1/SnykClient.java

```diff
@@ -55,7 +56,7 @@ public SnykClient(SnykConfig config) throws Exception {
     httpClient = builder.build();
   }
 
-  public SnykResult<NotificationSettings> getNotificationSettings(String org) throws java.io.IOException, java.lang.InterruptedException {
+  public SnykResult<NotificationSettings> validateCredentials(String org) throws java.io.IOException, java.lang.InterruptedException {
```

Comment:
Why this rename? It effectively returns the NotificationSettings, so I'd say the former name was ok?

Reply:
Ok, I'll put it back. You're right, but we're using it to validate credentials, and nothing is done with the notification settings, so I found it confusing.